Source/Initialization/ExternalField.cpp

```diff
@@ -172,11 +172,6 @@ ExternalFieldParams::ExternalFieldParams(const amrex::ParmParse& pp_warpx)
     //
     //  External fields from file
     //
-
-    if (E_ext_grid_type == ExternalFieldType::read_from_file ||
-        B_ext_grid_type == ExternalFieldType::read_from_file){
-            const std::string read_fields_from_path="./";
-            pp_warpx.query("read_fields_from_path", external_fields_path);
-    }
```

Comment:
TODO: add a comment saying that this is used both for external grid fields and external particles fields.